Gemfile.lock

```diff
@@ -168,12 +193,14 @@ GEM
     thread_safe (0.3.6)
     tzinfo (1.2.9)
       thread_safe (~> 0.1)
+    unicode-display_width (2.1.0)
     websocket-driver (0.7.5)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
 
 PLATFORMS
-  arm64-darwin-20
```

Comment:
I think we should leave this in here because we have someone who is using this platform.

Reply:
We are all using this platform, but we don't want the entire app to be locked to only using that platform

Reply:
This was in there initially because I prepended a bundle install command with arch -arm64e to see if it would prevent the wrong arch error that pops up. But it prevents anyone with a different platform from using our app.